scripts/pinned-tools.sh

```diff
@@ -111,6 +111,21 @@ function setAssemblyName() {
   ASSEMBLY_NAME="${1}_${GIT_HASH}"
 }
 
+# Uploads an assembly, given a $PREFIX and $PROJECT_NAME
+function uploadAssembly() {
+  if [ "$#" -ne 2 ]; then
+    echo "'uploadAssembly' expects two arguments."
+    echo "Example usage: 'uploadAssembly \$PREFIX \$PROJECT_NAME'"
```

Comment:
what does `\$PREFIX ` print? exactly that? (I honestly don't know)
though I think so far we have been using `<prefix>` to indicate which arguments are needed for a  command.

rename it to `<assembly prefix>` to make it clear what the prefix is for